fastlane/spec/env_spec.rb

```diff
@@ -4,9 +4,8 @@
 describe Fastlane do
   describe Fastlane::EnvironmentPrinter do
     before do
-      stub_request(:post, %r{https:\/\/refresher.fastlane.tools\/.*}).
```

Comment:
Is it possible to improve any of the tests here so that we could have caught the issue before releasing it?